NDTensors/src/dense/tensoralgebra/contract.jl

```diff
@@ -343,9 +347,10 @@ function _contract!(
   β::Number=zero(El),
 ) where {El,NC,NA,NB}
   tA = 'N'
+  t = TimerOutput()
```

Comment:
Reminder to remove this before we merge the PR.

Also, let's consider moving this `_contract!` function to `src/abstractarray/tensoralgebra/contract.jl`. That will help out with future work of removing the `Dense` type.